datasets/eurlex/README.md

```diff
@@ -0,0 +1,232 @@
+    - [Annotations](#annotations)
+      - [Annotation process](#annotation-process)
+      - [Who are the annotators?](#who-are-the-annotators)
+    - [Personal and Sensitive Information](#personal-and-sensitive-information)
+  - [Considerations for Using the Data](#considerations-for-using-the-data)
+    - [Social Impact of Dataset](#social-impact-of-dataset)
+    - [Discussion of Biases](#discussion-of-biases)
+    - [Other Known Limitations](#other-known-limitations)
+  - [Additional Information](#additional-information)
+    - [Dataset Curators](#dataset-curators)
+    - [Licensing Information](#licensing-information)
+    - [Citation Information](#citation-information)
+    - [Contributions](#contributions)
+
+## Dataset Description
+
+- **Homepage:** http://nlp.cs.aueb.gr/software_and_datasets/EURLEX57K/
+- **Repository:** http://nlp.cs.aueb.gr/software_and_datasets/EURLEX57K/
+- **Paper:** https://www.aclweb.org/anthology/P19-1636/
+- **Leaderboard:** N/A
+- **Point of Contact:** Ilias Chalkidis ( ihalk[at]aueb.gr )
+
+### Dataset Summary
+
+EURLEX57K can be viewed as an improved version of the dataset released by Mencia and Furnkranzand (2007), which has been widely used in Large-scale Multi-label Text Classification (LMTC) research, but is less than half the size of EURLEX57K (19.6k documents, 4k EUROVOC labels) and more than ten years old.
+EURLEX57K contains 57k legislative documents in English from EUR-Lex (https://eur-lex.europa.eu) with an average length of 727 words. Each document contains four major zones: 
+
+- the header, which includes the title and name of the legal body enforcing the legal act;
+- the recitals, which are legal background references; and
+- the main body, usually organized in articles.
+
+**Labeling / Annotation**
+
+All the documents of the dataset have been annotated by the Publications Office of EU (https://publications.europa.eu/en) with multiple concepts from EUROVOC (http://eurovoc.europa.eu/). 
+While EUROVOC includes approx. 7k concepts (labels), only 4,271 (59.31%) are present in EURLEX57K, from which only 2,049 (47.97%) have been assigned to more than 10 documents.  The 4,271 labels are also divided into frequent (746 labels), few-shot (3,362), and zero- shot (163), depending on whether they were assigned to more than 50, fewer than 50 but at least one, or no training documents, respectively.
+
+
+### Supported Tasks and Leaderboards
+
+The dataset supports:
+
+**Multi-label Text Classification:** Given the text of a document, a model predicts the relevant EUROVOC concepts.
+
+**Few-shot and Zero-shot learning:** As already noted, the labels can be divided into three groups: frequent (746 labels), few-shot (3,362), and zero- shot (163), depending on whether they were assigned to more than 50, fewer than 50 but at least one, or no training documents, respectively.
+
+### Languages
+
+All documents are written in English.
+
+## Dataset Structure
+
+### Data Instances
+
+
+```json
```

Comment:
same suggestions as for the other dataset card:
- use an actual example from python
- crop it if it's too long

Reply:
I used a actual example as loaded with `load_dataset`